src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/help-checkoutCredentialsId.html

```diff
@@ -1,3 +1,15 @@
 <div>
-    <p>Credentials used to check out sources during a build.</p>
+    <p>
+        Credentials used to check out sources during a build.
+    </p>
+    <p>
+        The default choice of "Same as scan credentials" is sufficient so long as you don't
+        need Jenkins to push some changes back into these repositories.
+    </p>
+    <p>
+        Otherwise you should create an <strong>SSH username with private key</strong> credential and
+        specify an SSH private key that belongs to the same user you've used for the scan
+        credentials. This causes Jenkins to check out source code over SSH, and therefore
+        you can run <code>git push</code> from your pipeline to push changes back.
```

Comment:
Would be better to move these two help files into `src/main/webapp/help/` and refer to them via an explicit `help` attribute on the `entry`s, so we do not have to duplicate content like this.